packages/@aws-cdk/aws-kinesis/lib/stream.ts

```diff
@@ -290,10 +291,26 @@ export class Stream extends StreamBase {
    * user's configuration.
    */
   private parseEncryption(props: StreamProps): {
-    streamEncryption?: CfnStream.StreamEncryptionProperty,
+    streamEncryption?: CfnStream.StreamEncryptionProperty | IResolvable
     encryptionKey?: kms.IKey
   } {
 
+    // if encryption properties are not set, default to KMS in regions where KMS is available
+    if (!props.encryption && !props.encryptionKey) {
+      const condition = new CfnCondition(this, 'Condition', {
+        expression: Fn.conditionOr(
+          Fn.conditionEquals(Aws.REGION, 'cn-north-1'),
+          Fn.conditionEquals(Aws.REGION, 'cn-northwest-1'))
+      });
+
+      const defaultEncryption = Fn.conditionIf(
+        condition.logicalId,
+        Aws.NO_VALUE,
+        { EncryptionType: 'KMS', KeyId: 'alias/aws/kinesis' });
+
+      return {streamEncryption: defaultEncryption, encryptionKey: undefined };
+      }
+
     // default based on whether encryption key is specified
     const encryptionType = props.encryption ??
     (props.encryptionKey ? StreamEncryption.KMS : StreamEncryption.UNENCRYPTED);
```

Comment:
I missed this from the original review - this should be indented one level (it's a continuation of the expression from the previous line):

```ts
    const encryptionType = props.encryption ??
      (props.encryptionKey ? StreamEncryption.KMS : StreamEncryption.UNENCRYPTED);
```

Reply:
done